tools/make_dictionary_file.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python2.7
+
+import sys
+"""
+Makes dictionary file from mozc
+(https://github.com/google/mozc/tree/master/src/data/dictionary_oss/dictionary*)
+
+The path to the output file: /personfinder/app/
```

Comment:
You don't need /personfinder/ because it's not always named personfinder in everyone's environment. But please include the exact file name.